ExeterTeam25/group-software-project
# Following/other profile features

What I did:

- Implemented following other users and viewing a feed with only your followed people's posts

To test:

- Check the followed user response tab is empty (as your user shouldn't have followed anyone yet)
- On the all user response tab, click on another account's name above their recent activity, this should take you to their profile page where you can then click the follow button
- The follow button should change its text to unfollow as you should now be following them
- On the followed user response tab their posts should now show
- If you unfollow them, the posts should no longer show
- If you go on your own profile page, the follow button should not show, instead there should be edit profile and delete profile buttons
